resources/gather-deploy-images.sh

```diff
@@ -26,9 +28,16 @@ function get-images-per-env() {
 	  fi
 	
 	  printf "%s\n" "$image"
-	done | sort -u > /tmp/all-images.txt
+	done | sort -u > "$IMAGES_FILE"
+
+	# For development purposes, allow injecting your own list of images
+	if [ -n "${MY_IMAGES_TO_VERIFY}" ]; then
```

Comment:
won't the script fail if MY_IMAGES_TO_VERIFY is unset (because of `set -u`)

Reply:
Thanks, good catch.

```suggestion
	if [ -n "${MY_IMAGES_TO_VERIFY:-}" ]; then
```

Reply:
Done in an upcoming revision.

Reply:
Actually maybe this could be removed entirely. It was useful when I was testing this in the tssc-jenkins repo, but perhaps it could be dropped for this repo.

Reply:
(Decided to remove it from the next revision.)